oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -144,7 +148,52 @@ public static ImpersonatedCredentials create(
   }
 
   /**
-   * @param sourceCredentials the source credential used as to acquire the impersonated credentials
+   * @param sourceCredentials the source credential used as to acquire the impersonated credentials.
```

Comment:
delete "as",
delete final period since it's not a complete sentence

Reply:
Done